snapshot/allowlist.js

```diff
@@ -8,7 +8,7 @@ async function main() {
     await ethers.getSigner()
   );
 
-  const elements = JSON.parse(fs.readFileSync("snapshot.json"));
+  const elements = JSON.parse(fs.readFileSync("./snapshot/snapshot.json"));
```

Comment:
@bennycio not sure what setup you have but when running the script from root, i had to make sure everything was prepended with `./snapshot`, especially since i stuck my `.env` in the root